packages/stream_video/lib/src/call/call.dart

```diff
@@ -1979,6 +1980,16 @@ class Call {
     return result.map((_) => none);
   }
 
+  Future<Result<bool>> setMultitaskingCameraAccessEnabled(bool enabled) async {
+    if (CurrentPlatform.isIos) {
+      final result =
+          await rtc.Helper.enableIOSMultitaskingCameraAccess(enabled);
+      return Result.success(result);
+    }
+
+    return const Result.success(false);
+  }
+
```

Comment:
`WebRTC.invokeMethod` can throw an exception, so if you already return a `Result` here it's good to wrap it in a try catch and return a failure for when the method channel fails.

Reply:
Good point